transformers/test_data/log_make.go

```diff
@@ -0,0 +1,64 @@
+package test_data
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"math/rand"
+)
+
+var (
+	logMakeRawLog = types.Log{
+		Address: common.HexToAddress(OasisAddresses()[0]),
+		Topics: []common.Hash{
+			common.HexToHash(constants.LogMakeSignature()),
+			common.HexToHash("0x00000000000000000000000000000000000000000000000000000000000c6280"),
+			common.HexToHash("0xcdd6659bca20e2b28ea10ead902280762ac8977c84459a152f90e561d50edf8c"),
+			common.HexToHash("0x0000000000000000000000006ff7d252627d35b8eb02607c8f27acdb18032718"),
+		},
+		Data:        hexutil.MustDecode("0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20000000000000000000000006b175474e89094c44da98b954eedeac495271d0f0000000000000000000000000000000000000000000000037d73b4e66fa60000000000000000000000000000000000000000000000000304f0cf4f5072f08000000000000000000000000000000000000000000000000000000000005e3e55fc"),
+		BlockNumber: 9440502,
+		TxHash:      common.HexToHash("0xb2f2f13a6ef0d1d6dd153d1a019929a31799498dcb1c100e04695fc0e95b9e58"),
+		TxIndex:     1772,
```

Comment:
Silly question, but where did you find the `TxIndex` and the log `Index`? I think I used to find them on etherscan but didn't see them recently.

Reply:
Yeah same found on Etherscan but think I'm doing it wrong 😅 

The transaction has a Nonce | Position field and I think we want to use Position for tx index (but I used Nonce, whoops!). 

And then the events on the transaction have a green circle to the left that I _think_ is the log index:
<img width="292" alt="Screen Shot 2020-04-14 at 2 15 13 PM" src="https://user-images.githubusercontent.com/7875345/79264600-7434df80-7e5a-11ea-9beb-b19d4be753d2.png">